src/utility-classes.css

```diff
@@ -0,0 +1,26 @@
+/*
```

Comment:
Nice, I think we can bring those in the Tegel repo and have it in the next release so we can use it ;)

Reply:
> Nice, I think we can bring those in the Tegel repo and have it in the next release so we can use it ;)

Yes, I agree!